curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63cf90d4696d8f00851873a4.md

```diff
@@ -23,6 +28,54 @@ Your `sport` property should be set to `Football`.
 assert.equal(myFavoriteFootballTeam.sport, 'Football');
 ```
 
+Your `myFavoriteFootballTeam` object should have a `year` property.
+
+```js
+assert.property(myFavoriteFootballTeam, 'year');
+```
+
+Your `year` property should be a number.
+
+```js
+assert.isNumber(myFavoriteFootballTeam.year);
+```
+
+Your `year` property should be set to `1986`.
+
+```js
+assert.equal(myFavoriteFootballTeam.year, 1986);
+```
+
+Your `myFavoriteFootballTeam` object should have an `isWorldCupWinner` property.
+
+```js
+assert.property(myFavoriteFootballTeam, 'isWorldCupWinner');
+```
+
+Your `isWorldCupWinner` property should be a boolean.
+
+```js
+assert.isBoolean(myFavoriteFootballTeam.isWorldCupWinner);
+```
+
+Your `isWorldCupWinner` property should be set to `true`.
+
+```js
+assert.equal(myFavoriteFootballTeam.isWorldCupWinner, true);
+```
+
+Your `myFavoriteFootballTeam` object should have a `headCoach` property.
+
+```js
+assert.property(myFavoriteFootballTeam, 'headCoach');
+```
+
+Your `headCoach` value should be an object.
```

Comment:
Kind of minor, but can we add a test or add to this test to make sure the object is empty?